shark-android/src/test/java/shark/LegacyHprofTest.kt

```diff
@@ -37,14 +37,14 @@ class LegacyHprofTest {
     val leak = analysis.applicationLeaks[0].leakTraces.first()
     assertThat(leak.leakingObject.className).isEqualTo("com.example.leakcanary.MainActivity")
     assertThat(leak.gcRootType).isEqualTo(GcRootType.STICKY_CLASS)
-    assertThat(analysis.allLeaks.sumBy { it.totalRetainedHeapByteSize!! }).isEqualTo(175112)
+    assertThat(analysis.allLeaks.sumBy { it.totalRetainedHeapByteSize!! }).isEqualTo(49584)
```

Comment:
The retained size was 175112 before due to a bug: GC roots were not counted as dominators, they are now. On this pre M heap dump, bitmap byte arrays were stored on the heap and referenced by both a native ref and a Bitmap instance. The bug made it look like it was referenced only by the bitmap, hence it being its dominator and that increasd the value (120Kb bitmap)